src/current/v24.1/cost-based-optimizer.md

```diff
@@ -359,18 +362,20 @@ For a join hint example, see [Use the right join type]({% link {{ page.version.v
 
 ### Supported join algorithms
 
-- `HASH`: Forces a hash join; in other words, it disables merge and lookup joins. A hash join is always possible, even if there are no equality columns - CockroachDB considers the nested loop join with no index a degenerate case of the hash join (i.e., a hash table with one bucket).
+- `HASH`: Forces a hash join; in other words, it disables merge and lookup joins. A hash join is always possible, even if there are no equality columns: CockroachDB considers the nested loop join with no index a degenerate case of the hash join (i.e., a hash table with one bucket).
 
 - `MERGE`: Forces a merge join, even if it requires re-sorting both sides of the join.
 
-- `LOOKUP`: Forces a lookup join into the right side; the right side must be a table with a suitable index. Note that `LOOKUP` can only be used with `INNER` and `LEFT` joins.
+- `LOOKUP`: Forces a lookup join into the right side; the right side must be a table with a suitable index. `LOOKUP` can only be used with `INNER` and `LEFT` joins.
 
-- `INVERTED`:  Forces an inverted join into the right side; the right side must be a table with a suitable [GIN index]({% link {{ page.version.version }}/inverted-indexes.md %}). Note that `INVERTED` can only be used with `INNER` and `LEFT` joins.
+- `INVERTED`: Forces an inverted join into the right side; the right side must be a table with a suitable [GIN index]({% link {{ page.version.version }}/inverted-indexes.md %}). `INVERTED` can only be used with `INNER` and `LEFT` joins.
 
     {{site.data.alerts.callout_info}}
     You cannot use inverted joins on [partial GIN indexes]({% link {{ page.version.version }}/inverted-indexes.md %}#partial-gin-indexes).
     {{site.data.alerts.end}}
 
+- `STRAIGHT`: Forces a straight join in the order specified in the query, without hinting a join algorithm. This can potentially override a more efficient query plan. A straight join that turns into another join type behaves as follows: hash or cross joins build and probe the right side; lookup joins probe an index on the right side; inverted joins probe an inverted index on the right side; and merge joins behave as standard merge joins. The join type is independent of whether a `INNER`, `LEFT`, or `RIGHT` straight join is specified.
```

Comment:
I think I would format the third sentence as several list items. "Merge joins behave as standard merge joins"?

Reply:
Going to leave this as I prefer the single block -- I think the latter details are sort of secondary to the main behavior, so I don't want to call too much attention to them.